### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by dylonChime**
## ℹ️ Description
doc update

### Type of change
documentation update only

## 🧪 How Has This Been Tested?
NA

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by hokyungh — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Unreleased
+[0.23.1] - 2023-04-20
```
Comment (georgezy-amzn) [inline CHANGELOG.md:1]: Missing `#`

**Review by georgezy-amzn — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-## Unreleased
+## [0.23.1] - 2023-04-20
```
Comment (linsang21) [inline CHANGELOG.md:1]: Same question as Android

Comment (dylonChime) [reply CHANGELOG.md:1]: Generated from MCM

Comment (alnlau) [reply CHANGELOG.md]: The date should be the day of the actual release

**Review by linsang21 — APPROVED** [review]

**Review by alnlau — APPROVED** [review]

**Pull request merged**